example/README.md

```diff
@@ -1,9 +1,12 @@
 # Example Setup
 
 0. Prerequisites:
-    - Python 3
-    - Docker and `docker-compose`
+    - Python 3.8+
+    - docker compose V2
     - ~16GiB of free storage
-2. Launch the containers: `docker-compose up -d`
+1. Clear local data dir if exists: `rm -vr data`
+2. Launch containers: `docker compose up -d`
+    - Check permissions for subdirectory `./data` since there goes the load
 3. Download and ingest the documents into Solr: `./ingest.py`
-4. Search the corpus: `http://localhost:8181`
\ No newline at end of file
+    - If only interested in books: `./ingest_books_only.py`
```

Comment:
I'd prefer a `--books-only` flag to the `ingest.py` script instead of creating a second one that shares much of the code with the existing script.